src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -10759,6 +10759,156 @@ static void M()
                 matchingFilters: new List<CompletionFilter> { FilterSet.LocalAndParameterFilter });
         }
 
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task CompletionWithSemicolonForMethod()
```

Comment:
Could you add a test where the invocation is an inner expression? e.g. assuming there's a method Bar return int and trigger completion in `Foo(Bar$$)`

Reply:
Sure